### PR TITLE
PTA's Woe lobby screen

### PR DESCRIPTION
**Pull request by widgetbeck**
**Changelog**

:cl:
- add: Added new shittle lobby art.

### CODE REVIEW

**Comment by widgetbeck** [comment]
shittle_render_full

**Comment by kipdotnet** [comment]
is that the fucking vgroid on the shuttle console lmao

**Comment by widgetbeck** [comment]
nah its cog but honestly....... it should be the vgroid....

**Comment by hivehum** [comment]
beck do you want to change it to vgroid or am i good to merge this

**Comment by hivehum** [comment]
oh you also forgot to edit attributions.yml

**Comment by widgetbeck** [comment]
you're good to merge it after i fix attributions

**Comment by widgetbeck** [comment]
we good

File: Resources/Textures/_Impstation/LobbyScreens/attributions.yml [hunk 1]
```diff
@@ -37,3 +37,8 @@
   license: "CC-BY-SA-4.0"
   copyright: "snip cowlegz"
   source: "https://github.com/impstation/imp-station-14/"
+
+- files: ["PTAs-woe.webp]
```
Comment (hivehum) [inline Resources/Textures/_Impstation/LobbyScreens/attributions.yml:41]: idk how to push commits to other peoples prs. you missed a "

**Review by hivehum — APPROVED** [review]
yayyyyyyy

**Pull request merged**